src/js/tech/html5.js

```diff
@@ -767,7 +770,11 @@ class Html5 extends Tech {
       return super.removeRemoteTextTrack(track);
     }
 
-    var tracks, i;
+    let tracks, i;
+
+    // TODO: track can be html text element due to addRemoteTextTrack—consider refactoring
+    let trackElement = this.remoteTextTrackEls().getTrackElementByTrack_(track);
```

Comment:
This confused me quite a bit (also due to the removeTrack_ bug), but `addRemoteTextTrack` returns an `HTMLTrackElement` or `{track: TextTrack}`—neither of which are `TextTrack` capable of being removed.